client/app/components/manager-navbar/manager-navbar-menu/manager-navbar-menu.html

```diff
@@ -0,0 +1,37 @@
+<!-- <div class="oui-navbar-menu oui-navbar-menu_fixed oui-navbar-menu_right"> -->
+    <header class="oui-navbar-menu__header">
+        <oui-back-button class="oui-navbar-menu__back"
+            on-click="$ctrl.toggleMenu($ctrl.menuName, true)">
```

Comment:
What about `data-` prefix?

Reply:
It's a custom html element, not necessary :)